gnomad_qc/v4/annotations/Dockerfile_vrs084_v4

```diff
@@ -53,6 +58,17 @@ RUN wget https://github.com/vcftools/vcftools/releases/download/v${VCFTOOLS_VERS
 	&& make install \
 	&& make clean
 
+# Install bgzip
```

Comment:
It might be better to have this comment say `# Install bgzip from htslib`

As someone with no familiarity with htslib, this looked like a typo, since there wasn't anything in the install commands that mentioned bgzip!